client/rest/src/main/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumer.java

```diff
@@ -67,6 +67,13 @@ public HeapBufferedAsyncResponseConsumer(int bufferLimit) {
         this.bufferLimit = bufferLimit;
     }
 
+    /**
+     * Get the limit of the buffer.
+     */
+    public int getBufferLimit() {
```

Comment:
Ahh I totally missed that this was client only, `getBufferLimitBytes` makes sense